config/routes.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
```

Comment:
It would be nice when we be had friendly Urls.
https://github.com/kaminari/kaminari#creating-friendly-urls-and-caching

Reply:
Ok will do :)